Formula/h/helm-ls.rb

```diff
@@ -0,0 +1,22 @@
+class HelmLs < Formula
+  desc "Language server for Helm"
+  homepage "https://github.com/mrjosh/helm-ls"
+  url "https://github.com/mrjosh/helm-ls.git",
+      tag:      "v0.0.9",
+      revision: "5dbde8a5d83a7cd3c21e55e8150100a2f7dd462e"
+  license "MIT"
+  head "https://github.com/mrjosh/helm-ls.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    system "make"
+    bin.install "bin/helm_ls"
+
+    generate_completions_from_executable(bin/"helm_ls", "completion")
+  end
+
+  test do
+    system bin/"helm_ls", "version"
```

Comment:
```suggestion
    assert_match version.to_s, shell_output(bin/"helm_ls version")
```

Reply:
We will need an additional test that exercises some functionality of the software. Versions or help checks rarely fail even if there are compilation issues. In this case it's worth keeping to ensure the ldflags are set correctly.